v20.2/disaster-recovery.md

```diff
@@ -0,0 +1,337 @@
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>2 or More Regions</b></td>
+      <td style="color:#FF0000"><b>X</b></td>
+      <td>Cluster will be unavailable. <br><br>Potential data loss between last backup and time of outage if region and servers did not come back online. This would be a considerable disaster (i.e., 2 or more data centers destroyed).</td>
+      <td>When the regions comes back online, try <a href="cockroach-start.html">restarting the nodes</a> in the cluster. <br><br>If the regions do not come back online and servers are lost or destroyed, try <a href="restore.html">restoring the latest cluster backup</a> into a new cluster.</td>
+    </tr>
+  </tbody>
+</table>
+
+<span style="color:#228B22"><b>√</b></span> = Available
+<span style="color:#FF0000"><b>X</b></span> = Outage
+
+## Data failure
+
+When dealing with data failure due to bad actors, rogue applications, or data corruption, domain expertise is required to identify the affected rows and determine how to to remedy the situation (e.g., remove the incorrectly inserted rows, insert deleted rows, etc.). However, there are a few options for short term remediation that you can take:
+
+#### Basic recovery actions
+
+- If you are an enterprise user, use your backup file to [restore to a point in time](backup-and-restore-advanced-options.html#point-in-time-restore) to a point where you are certain there was no corruption.
+
+    {{site.data.alerts.callout_success}}
+    Instead of dropping the corrupted table or database, we recommend [renaming the table](rename-table.html) or [renaming the database](rename-database.html) so you have historical data to compare to later.
+    {{site.data.alerts.end}}
+
+- If you are within the [garbage collection window](configure-replication-zones.html#replication-zone-variables) (default is 25 hours), run [`AS OF SYSTEM TIME`](as-of-system-time.html) queries and use [`CREATE TABLE AS … SELECT * FROM`](create-table-as.html) to create comparison data and run “diffs” to find the offending rows to fix. If you are outside of the garbage collection window, you will need to use a backup to run comparisons.
+
+    {{site.data.alerts.callout_info}}
+    Instead of dropping the corrupted table or database, we recommend [renaming the table](rename-table.html) or [renaming the database](rename-database.html) so you have historical data to compare to later.
+    {{site.data.alerts.end}}
+
+#### Advanced recovery actions
+
+- While you clean up the data, put your application in “read only” mode and only run [`AS OF SYSTEM TIME`](as-of-system-time.html) queries from the application.
+- If your cluster is running and you do not have a backup that encapsulates the time you want to restore to, immediately trigger a new [backup `with_revision_history`](backup-and-restore-advanced-options.html#backup-with-revision-history-and-point-in-time-restore) and you will have a backup you can use to restore to the desired point in time.
```

Comment:
You really have two options here--trigger backup `with_revision_history` OR backdate your backup to the `AS OF SYSTEM TIME` you want to restore to. The second option is important because it'll be available to CORE users with our forthcoming "make full backup core" change in 20.2

Should we also label CORE vs ENTERPRISE options here?